x-pack/test/functional/apps/maps/discover.js

```diff
@@ -17,7 +17,6 @@ export default function({ getService, getPageObjects }) {
 
     it('should link geo_shape fields to Maps application', async () => {
       await PageObjects.discover.selectIndexPattern('geo_shapes*');
-      await PageObjects.discover.clickFieldListItem('geometry');
```

Comment:
Should this call be replaced with calls to `PageObjects.discover.findFieldByName` like in the discover tests?

Reply:
Actually, that's not really necessary (even in discover tests), `findFieldByName` only types the field name in the search bar, to make a field to be in top of sidebar. 
This was done in first attempt to fix flaky tests, assuming that failing was caused by a field was too low and goes out of visible screen.
Since that time `findFieldByName` stayed in tests, don't see a problem with that. @dmlemeshko what are your thoughts?

Reply:
If these tests are not flaky and do not require pre-search (flaky-test-runner showed 42 times passed), I think we don't need it..